packages/ndla-audio-search/src/AudioSearchList.tsx

```diff
@@ -47,10 +45,15 @@ export default function AudioSearchList({
     return <p>{translations.noResults}</p>;
   }
   if (searching && !((audios?.length ?? 0) > 0)) {
-    return <StyledResultSpinner />;
+    return (
+      <SpinnerWrapper>
+        <Spinner />
+      </SpinnerWrapper>
+    );
   }
+
   return (
-    <div>
+    <ul>
```

Comment:
La oss sette list-style none her

Reply:
hvorfor er det nødvendig? bare ut av interesse altså

Reply:
Hvis vi ikke setter det er det en viss fare for at listeelementene plutselig får bullets dersom vi endrer noe der. Tenkte det var like greit å være føre var da.